### PR TITLE
feat: release rdbms as beta

### PR DESCRIPTION
**Pull request by moT01**
Replaces https://github.com/freeCodeCamp/freeCodeCamp/pull/45405 - that one just got a little too messy after reverting and then adding back translations and stuff, so I recreated it
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by ShaunSHamilton — DISMISSED** [review]
I am happy with this 👍

File: curriculum/utils.js [hunk 1]
```diff
@@ -25,19 +25,19 @@ const superBlockToOrder = {
...
+  'machine-learning-with-python': 9,
+  'coding-interview-prep': 10,
+  'relational-database': 12
 };
 
 const superBlockToNewOrder = {
   ...superBlockToOrder,
-  '2022/responsive-web-design': 12
+  '2022/responsive-web-design': 11
 };
```
Comment (nhcarrigan) [inline curriculum/utils.js:41]: Should we not have `relational-database` in the new order instead? Or do we want to display it on all languages?

Comment (moT01) [reply curriculum/utils.js:41]: note added

**Comment by moT01** [comment]
I added the note we discussed in chat - in shows up on all but english RDBMS pages like this:
Screen Shot 2022-03-16 at 3 08 24 PM

**Comment by QuincyLarson** [comment]
@moT01 I like this message copy. I think we should put it at the very top of the view, and have it display in the currently selected language (if there's a way we can get it into the Crowdin's "Learn User Interface" section). Then we can safely include this certification in all language versions, and we can use this same approach going forward for yet-untranslated certifications.

**Comment by moT01** [comment]
It looks like this now:
Screen Shot 2022-03-17 at 9 27 40 AM
The top message shows on all languages except English. It will be added to crowdin after this PR is merged, so the message won't be translated until it goes through that process.

File: curriculum/utils.test.ts [hunk 2]
```diff
@@ -31,14 +31,14 @@ describe('getSuperOrder', () => {
...
-    expect(getSuperOrder('machine-learning-with-python')).toBe(10);
-    expect(getSuperOrder('coding-interview-prep')).toBe(11);
+    expect(getSuperOrder('back-end-development-and-apis')).toBe(4);
+    expect(getSuperOrder('quality-assurance')).toBe(5);
+    expect(getSuperOrder('scientific-computing-with-python')).toBe(6);
+    expect(getSuperOrder('data-analysis-with-python')).toBe(7);
+    expect(getSuperOrder('information-security')).toBe(8);
+    expect(getSuperOrder('machine-learning-with-python')).toBe(9);
+    expect(getSuperOrder('coding-interview-prep')).toBe(10);
+    expect(getSuperOrder('relational-database')).toBe(12);
```
Comment (ojeytonwilliams) [inline curriculum/utils.test.ts:41]: I see this is quite annoying to change.  I'll try to come up with something a bit less irritating.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by ShaunSHamilton — APPROVED** [review]
ADTFAIAGTGTPR 👍

**Pull request merged**